crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -155,19 +155,31 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
-    /// Returns an [`EntityCommands`] for the given `entity` (if it exists) or spawns one if it
-    /// doesn't exist. This will return [`None`] if the `entity` exists with a different generation.
+    /// Pushes a [`Command`] to the queue for creating the given [`Entity`] if it does not exist,
+    /// and returns its corresponding [`EntityCommands`].
+    ///
+    /// Returns `None` if the entity does not exist.
```

Comment:
These docs are unclear to me still. It's supposed to spawn the entity if it doesn't exist; why is returning `None` here useful?

Reply:
Oops, my bad. It should say that it returns `None` if there is already an entity with the same ID.